Patches/Core/ThingDefs_Races/Races_Animal_Bears.xml

```diff
@@ -227,14 +227,14 @@
 	<Operation Class="PatchOperationAdd">
 		<xpath>Defs/ThingDef[defName="Bear_Polar"]/race</xpath>
 		<value>
-			<baseBodySize>1.55</baseBodySize>
```

Comment:
For reference, a human has a size of 1, and a rhino has 3, and an elephant has size 4. I don't know if this is new/useful information to you.